src/s3ql/backends/gs.py

```diff
@@ -451,29 +488,14 @@ def write_fh(self, fh, key: str, md5: bytes,
         fh.seek(0)
 
         md5_run = hashlib.md5()
-        try:
-            self.conn.write(body_prefix)
-            while True:
-                buf = fh.read(BUFSIZE)
-                if not buf:
-                    break
-                self.conn.write(buf)
-                md5_run.update(buf)
-            self.conn.write(body_suffix)
-        except ConnectionClosed:
```

Comment:
@Nikratio This logic seemed to be copy pasted and not make much sense, so I just removed it.

Reply:
Without this logic, we will get a generic dugong exception that tells us nothing beyond "writing to the server failed". This code tries to read a server response, which allows us to raise an exception with more details provided by the server.

Reply:
I can't find where in the code the response is used? at the call site it is ignored.